SDKAndroid/src/test/java/app/cybrid/sdkandroid/components/KYCViewTest.kt

```diff
@@ -0,0 +1,67 @@
+package app.cybrid.sdkandroid.components
```

Comment:
What about negative testing? How can this view fail? What about services not loading data?